src/components/Avatar/Avatar.story.js

```diff
@@ -0,0 +1,357 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs/react';
+import Avatar from './Avatar';
+import Tooltip from '../Tooltip/Tooltip';
+import storyWrapper from '../../storybook-addons/storyWrapper';
+
+const stories = storiesOf('Avatar.Avatar', module);
+
+stories.addDecorator(withKnobs);
+
+stories.add(
+    'Overview',
+    storyWrapper(
+        'The avatar is a round user profile icon. The avatar\'s primary role is as a button.',
+        <Avatar
+            name={text('Name', 'David Smith')}
+            initials={text('Initials', 'DS')}
+            size={select('Size', {
+                xs: 'XS', sm: 'SM', md: 'MD', lg: 'LG',
+            }, 'md')}
+            hasHalo={boolean('Halo', false)}
+            hasNotification={boolean('Notification', false)}
+        />,
+        <div>
+            <Tooltip tooltip="Click me!">
+                <Avatar
+                    name="Harriet Inkwell"
+                    initials="HI"
+                    onClick={() => { alert('hello!'); }} // eslint-disable-line
+                    style={{ marginRight: '30px' }}
+                />
+            </Tooltip>
+            <Avatar
+                name="David Smith"
+                src="https://randomuser.me/api/portraits/men/3.jpg"
+                style={{ marginRight: '30px' }}
+            />
+            <Avatar
+                name="Jane Smith"
+                src="https://randomuser.me/api/portraits/women/58.jpg"
+                hasHalo
+                hasNotification
```

Comment:
@mattpauldavies If this is enough to enable the prop, then would we need to change the props table? Currently the props table shows:

property | value | required | default
------------|---------|------------|----------
hasHalo | bool | no | false

But the above implementation is a bit confusing, because the prop has no value but defaults to true

Reply:
This is the React preferred implementation (and in fact enforced by our linter).  It's based on the HTML spec...

`hasHalo` without assignation is not `null` it is `true`

The following would be no value: *(and invalid, per the prop-type validation)*

```
<Avatar hasHalo="" ... />
<Avatar hasHalo={null} ... />
```

This is valid for turning off the halo:

```
<Avatar ... /> // uses default which is false
<Avatar hasHalo={false} ... />
```

This is valid for turning on the halo:  *(these are equivalent)*

```
<Avatar hasHalo ... />
<Avatar hasHalo={true} ... />
```

 I hope that makes sense!